astro-airflow-iris/{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 dynamic = ["dependencies", "version"]
 
 [project.scripts]
-{{ cookiecutter.repo_name }} = "{{ cookiecutter.python_package }}.__main__:main"
+"{{ cookiecutter.repo_name }}" = "{{ cookiecutter.python_package }}.__main__:main"
```

Comment:
Do the quotes not change any of the behaviour? We don't have these in the base template either.

Reply:
I have the same question. Is this a correct fix, it looks like `ruff` is treating this as `pyproject.toml` but it is using jinja syntax that ruff will not understand.

Reply:
It does not change the behavior from what I'm seeing here. I've tested it by creating a new project with a starter and using this branch as the `--checkout` argument. It resolves the repo name correctly and `kedro run` works as well.

Reply:
I tested it locally as well and it does seem to work fine. I specifically checked that the resulting `pyproject.toml` file looked the same as when using the released version.